core/src/i18n/en.yaml

```
@@ -433,6 +433,15 @@ cron-jobs:
   not-scheduled-yet: None scheduled yet
   schedule: Schedule
   title: CronJobs
+service-accounts:
+  title: Service Accounts
+  auto-mount-token:
+    enabled: enabled
+    disabled: disabled
+    description: Automount is discouraged, as a Pod with injected token will have the same permissions as the Service Account.
+  headers:
+    auto-mount-token: Auto-mount token
```

```suggestion
    auto-mount-token: Automount Token
```
Headers are capitalized. Not sure why automount was written differently than elsewhere?